.rubocop.yml

```diff
@@ -10,14 +10,16 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 25
```

Comment:
- [ ] **Optional:** Rather than edit the `.rubocop.yml` file, find ways to make your methods shorter.